### PR TITLE
Feature server and frontend svc annotations

### PR DESCRIPTION
**Pull request by lamc**
@rajdas98 
@ispeakc0de
@Jasstkn 

#### What this PR does / why we need it:
Adds capability to add service annotations for litmusportal-frontend such as load balancer resource tags or type when installing LitmusChaos with LoadBalancer (as shown below)
```
service.beta.kubernetes.io/aws-load-balancer-additional-resource-tags
service.beta.kubernetes.io/aws-load-balancer-type
```

As well as adds capability to add service annotations for litmusportal-server 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - NA

#### Checklist
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by Jasstkn — APPROVED** [review]
Hi. Thank you for your contribution! 
LGTM 🎉

**Comment by Jasstkn** [comment]
@lamc hi. could you update from main branch?

**Comment by lamc** [comment]
@Jasstkn I pulled from master, how would you like for me to proceed re the conflicts that came from pulling master branch? additionally, I see that my commits are signed off -- any suggestions?

**Comment by Jasstkn** [comment]
> @Jasstkn I pulled from master, how would you like for me to proceed re the conflicts that came from pulling master branch? additionally, I see that my commits are signed off -- any suggestions?

pick up from master

**Comment by lamc** [comment]
@Jasstkn got it, I pulled latest from master and fixed conflicts

**Comment by Jasstkn** [comment]
> Details

please, bump chart version & run helm-docs.

**Comment by lamc** [comment]
> > Details
> 
> please, bump chart version & run helm-docs.

@Jasstkn done!

**Comment by Jasstkn** [comment]
> > > Details
> > 
> > 
> > please, bump chart version & run helm-docs.
> 
> @Jasstkn done!

DCO doesn't pass :(

**Comment by lamc** [comment]
@Jasstkn all checks have passed!! 🎉 Please review 🙏

**Review by uditgaurav — APPROVED** [review]

File: charts/litmus/README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ $ helm install litmus-portal litmuschaos/litmus
 | ingress.host.paths.frontend | string | `"/(.*)"` | You may need adapt the path depending your ingress-controller |
 | ingress.name | string | `"litmus-ingress"` |  |
 | ingress.tls | list | `[]` |  |
+| minikube | bool | `false` |  |
```
Comment (gdsoumya) [inline charts/litmus/README.md:57]: This is not needed, was removed in master

File: charts/litmus/values.yaml [hunk 2]
```diff
@@ -284,3 +286,5 @@ openshift:
     name: litmus-portal
     annotations: {}
     host: ""
+
+minikube: false
```
Comment (gdsoumya) [inline charts/litmus/values.yaml:290]: This is not needed, was removed in master

**Review by gdsoumya — CHANGES_REQUESTED** [review]

**Review by gdsoumya — APPROVED** [review]

**Comment by lamc** [comment]
Thank you @Jasstkn @gdsoumya @uditgaurav for reviewing and responding so quickly! it was a pleasure working with ya'll 😄

**Comment by Jasstkn** [comment]
> Thank you @Jasstkn @gdsoumya @uditgaurav for reviewing and responding so quickly! it was a pleasure working with ya'll 😄

Can't wait for your next contribution! You did well =)

**Comment by gdsoumya** [comment]
> Thank you @Jasstkn @gdsoumya @uditgaurav for reviewing and responding so quickly! it was a pleasure working with ya'll smile

Thanks for the contribution! Looking forward to more!

**Pull request merged**